web-project/src/main/java/com/epam/webproject/dao/impl/AuthorDaoImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.webproject.dao.impl;
+
+import java.util.Collection;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Query;
+
+import org.springframework.stereotype.Repository;
+
+import com.epam.webproject.dao.AuthorDao;
+import com.epam.webproject.domain.Author;
+
+@Repository
+public class AuthorDaoImpl implements AuthorDao {
+
+    @PersistenceContext
+    private EntityManager entityManager;
+
+    @Override
+    public Author getById(int id) {
+        return null;
+    }
+
+    @Override
+    public Author save(Author author) {
```

Comment:
This method is not used due to the poor ui. All unused methods are removed from the repository layer.